dashboard/urls.py

```diff
@@ -45,28 +45,28 @@
     url(r'^courses/(?P<course_id>[0-9]+|)/?view_file_access_within_week', login_required(TemplateView.as_view(template_name='view_file_access_within_week.html')), name="view_file_access_within_week"),
```

Comment:
All of these url's that reference the removed *.html files can also be removed

Reply:
Could all the files under `dashboard/static` be removed as well?

Reply:
Yeah, if you have it all somewhere else. The 
dashboard/static/json is all old and unused.
dashboard/static/images were the images that we might want to retain if they fit into the design
dashboard/static/js had the code for creating the d3 for the views that you've probably re-created. 
dashboard/static/css I'm guessing isn't needed.

Reply:
The images are in `assets/src/static`, so I think the `dashboard/static` folder can be removed then.

Reply:
I've removed these files now.